utils/colors.css

```diff
@@ -0,0 +1,9 @@
+:root{
+    --background-color: #E5E5E5;
+    --navBar-color:#FFFFFF;
+    --primary-color:#C4C4C4;
+    --btn-primary-color:#9A9A9A;
+    --btn-secondary-color:#FFFFFF;
+    --btn-focus-color:#3B82F6;
+    --stdWhite-color:#FFFFFF;
+}
```

Comment:
#FFFFFF is repeated. You can remove the repeated ones.

Reply:
done